src/main/java/io/micronaut/build/RunMojo.java

```diff
@@ -459,22 +538,20 @@ private void runAotIfNeeded() {
         }
     }
 
-    private String findJavaExecutable() {
-        String executable;
-        Toolchain toolchain = this.toolchainManager.getToolchainFromBuildContext("jdk", mavenSession);
-        if (toolchain != null) {
-            executable = toolchain.findTool(JAVA);
-        } else {
-            File javaBinariesDir = new File(new File(System.getProperty("java.home")), "bin");
-            if (Os.isFamily(Os.FAMILY_UNIX)) {
-                executable = new File(javaBinariesDir, JAVA).getAbsolutePath();
-            } else if (Os.isFamily(Os.FAMILY_WINDOWS)) {
-                executable = new File(javaBinariesDir, "java.exe").getAbsolutePath();
-            } else {
-                executable = JAVA;
-            }
+    private void maybeStartTestResourcesServer() {
+        try {
+            executorService.executeGoal(THIS_PLUGIN, MicronautStartTestResourcesServerMojo.NAME);
+        } catch (MojoExecutionException e) {
+            getLog().error(e);
+        }
+    }
+
+    private void maybeStopTestResourcesServer() {
+        try {
+            executorService.executeGoal(THIS_PLUGIN, MicronautStartTestResourcesServerMojo.NAME);
```

Comment:
:x: Looks like this should be `MicronautStopTestResourcesServerMojo.NAME`

Reply:
Nice catch! I missed it while reviewing your PR, sorry about that. Fixed now.

Reply:
It actually makes me wonder why it worked :D